server.js

```diff
@@ -142,8 +170,9 @@ app.post("/slack_response", function(req, res) {
   var messageTime = new Date(strReq.message_ts * 1000);
   var actionTime = new Date(strReq.action_ts * 1000);
 
+  userID = strReq.user.id;
```

Comment:
What's this doing here? What got changed?

Reply:
I did realize that, I think it was when you fixed the userID not defined error, my fork didn't update with the change. So it marked it as if I changed it, it's in the same spot. Nothing technically changed. Git just saw it as new for my fork. I can fix it if you'd like, but it'll have no effect on the actual code. It just won't be highlighted green.

Reply:
Hmm, I'd prefer to keep it how it is, seeing as I know it works

Reply:
What do you mean? I didn't change that line. It's the same as it currently is. It's just highlighted as a change because my fork was old. I can create a new pull request where it isn't highlighted if you'd like.